src/components/standalone/users_objects/CreateOrEditHostSetDrawer.vue

```diff
@@ -105,7 +106,18 @@ watch(
   }
 )
 
+// compute redirectItemsName the name of the portforward rule using this object
+watch(
+  () => props.currentHostSet?.matches,
+  async () => {
+    if (props.currentHostSet?.matches) {
```

Comment:
We need an `else` branch with `redirectItemsName.value = ''`. Try to test this:
- Trigger the validation error about IP range
- Create a new host set containing an IP range
- Even if the new host set is not used by any port forward, the validation error appears again